bin/verify-era-proof-attestation/src/verification.rs

```diff
@@ -100,16 +135,21 @@ fn is_quote_matching_policy(
         );
         return false;
     }
-
-    check_policy(
-        attestation_policy.sgx_mrsigners.as_deref(),
-        &quote.report_body.mrsigner,
-        "mrsigner",
-    ) && check_policy(
-        attestation_policy.sgx_mrenclaves.as_deref(),
-        &quote.report_body.mrenclave,
-        "mrenclave",
-    )
+    match &quote.report {
+        Report::SgxEnclave(report_body) => {
+            check_policy(
+                attestation_policy.sgx_mrsigners.as_deref(),
+                &report_body.mr_signer,
+                "mrsigner",
+            ) && check_policy(
+                attestation_policy.sgx_mrenclaves.as_deref(),
+                &report_body.mr_enclave,
+                "mrenclave",
+            )
+        }
+        Report::TD10(_) => false,
+        Report::TD15(_) => false,
```

Comment:
Is it just a `TODO`, or does it work fundamentally differently for TDX?

Reply:
well, it's a TODO for the while binary, as it only requests "sgx" proofs

Reply:
we would have to extend the tool to accept valid `mrtd` and `rtmr[0..3]`

Reply:
I’ve never heard of `mrtd` and `rtmr`. I think it’s something specific to TDX. Just leaving this here for reference in case I ever need to revisit it: ttps://www.kernel.org/doc/Documentation/x86/tdx.rst
```
The TDX module records the state of the TDX guest in various stages of
the guest boot process using the build time measurement register (MRTD)
and runtime measurement registers (RTMR). Measurements related to the
guest initial configuration and firmware image are recorded in the MRTD
register. Measurements related to initial state, kernel image, firmware
image, command line options, initrd, ACPI tables, etc are recorded in
RTMR registers. For more details, as an example, please refer to TDX
Virtual Firmware design specification, section titled "TD Measurement".
At TDX guest runtime, the attestation process is used to attest to these
measurements.
```